src/js/espn-borischentiers.js

```diff
@@ -31,15 +31,47 @@ window.addEventListener('load', function () {
   let promises = Object.keys(links).map(pos => setTierInfo(pos));
   Promise.all(promises)
     .then(_responses => {
-      addTierHeader();
-      const rows = document.getElementsByClassName(TABLE_BODY_CLASS)[0].rows;
-      for (let row of rows) {
-        addTierTd(row);
-      }
+      widenTable();
+      addBorisChenTiersHTML();
+      addRefreshButton();
     })
     .catch(_e => { });
 });
 
+function widenTable() {
+  const tableWrapper = document.getElementsByClassName(CONTAINER_CLASS)[0];
+  tableWrapper.style.width = '105%';
+}
+
+function addRefreshButton() {
+  const playersTableControls = document.getElementsByClassName(PLAYERS_TABLE_CONTROLS_CLASS)[0];
+  let newButton = document.createElement('button');
+  newButton.innerHTML = 'Refresh';
```

Comment:
Should we call this 'Refresh Boris Chen Tiers' or something to make it more descriptive? Because it won't refresh the other data in the table, so it seems a bit misleading.